hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/health/NodeDataProvider.java

```diff
@@ -0,0 +1,31 @@
+package pl.allegro.tech.hermes.management.domain.health;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+@Component
+class NodeDataProvider {
```

Comment:
We have something similar: https://github.com/allegro/hermes/blob/master/hermes-common/src/main/java/pl/allegro/tech/hermes/common/util/InetAddressHostnameResolver.java. Maybe you can merge those two implementations?

Reply:
I will not change it.